### DISCUSSION TITLE
Adding a new line before a new prompt

### DISCUSSION THREAD

**Discussion by ghost** [reobin/typewritten#141] [Q&A] [answered]
Hello! Is it possible to add a new line before every prompt so I can clearly see where the prompt begins?

**Comment by reobin** [comment, marked as answer]
Hello @srrvnn-g ! ☺️

All "multi-line" layouts use an empty line above the prompt. This includes `half_pure`, `pure`, `pure_verbose` and `multiline`. Just set your `TYPEWRITTEN_PROMPT_LAYOUT` variable to one of these values in your `.zshrc`.

For single-line layouts, the line is not available as an option. Please create an issue about it if you'd like, I'll be happy to discuss the option further.
